From 4592ca10a7cc8d6ae9cc61d4ef744c2ffb248f9d Mon Sep 17 00:00:00 2001
From: Jarod Wilson <jarod@redhat.com>
Date: Fri, 19 Dec 2014 07:20:31 -0500
Subject: [redhat] configs: enable overlayfs

Message-id: <1418973631-52031-1-git-send-email-jarod@redhat.com>
Patchwork-id: 102364
O-Subject: [RHEL7 PATCH] redhat/configs: enable overlayfs
Bugzilla: 985875
RH-Acked-by: Steve Best <sbest@redhat.com>

Bugzilla: 985875

The set for overlayfs neglected to add a config file for it, so the
patches as submitted don't actually enable overlayfs, it defaults to off.

CC: David Howells <dhowells@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_OVERLAY_FS b/redhat/configs/generic/CONFIG_OVERLAY_FS
new file mode 100644
index 0000000..b30428d
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_OVERLAY_FS
@@ -0,0 +1 @@
+CONFIG_OVERLAY_FS=m
-- 
1.7.1